torch/functional.py

```diff
@@ -95,28 +96,26 @@ def btriunpack(LU_data, LU_pivots, unpack_data=True, unpack_pivots=True):
         >>> A_ = torch.bmm(P, torch.bmm(A_L, A_U))
     """
 
-    nBatch, sz, _ = LU_data.size()
+    sz = LU_data.size(-1)
```

Comment:
Dumb question: doesn't this return the size of the last dimension (whereas previously `sz` was the size of the next to last dimension?)

Reply:
(Is this because the input is square outside of the batch dimension? Maybe we can add that to the docs. Sorry, I have never used btriunpack ever in my life XD )

Reply:
Yep, LU_data is a batch of square inputs.